qemu/tests/cfg/smbios_default_check.cfg

```diff
@@ -0,0 +1,24 @@
+- smbios_default_check: install setup image_copy unattended_install.cdrom
+    only q35
+    no Host_RHEL.m6,Host_RHEL.m7,Host_RHEL.m8.u0,Host_RHEL.m8.u1
+    # dmidecode not available (and no plans for it) on s390
+    no s390x
+    type = smbios_default_check
+    requires_root = yes
+    start_vm = yes
+    kill_vm = yes
+    # Expected output
+    System_Manufacturer = "^Red Hat$"
+    System_SKU_Number = "^[89].[23456789].0$"
```

Comment:
I guess it could be other number besides '0', like '8.2.1'?

Reply:
Explain it with x.y.z, x should be 8 or 9, if x is equal to 8, y should be greater than 2, if x is equal to 9, y can be one of 0-9, z can only be 0.
And I wlii change the code to `System_SKU_Number = "^8.[23456789].0|9.\d.0$"` ,do you agree?

Reply:
@leidwang If you are sure that z can only be '0', then it's fine.

Reply:
@vivianQizhu Ok, I'm sure, z can only be '0'. Thanks!